### PR TITLE
Move Cloudron into unofficial method

### PR DESCRIPTION
**Pull request by gramakri**
We do not offer hosting services and don't support lounge itself directly. So, we prefer not be listed under services but as an unofficial install method. We do maintain the lounge package though (I think for the last 4 years now). Thanks!

### CODE REVIEW

File: _docs/install-and-upgrade.md [hunk 1]
```diff
@@ -202,6 +202,7 @@ supported (even when hosted on this website), so use them at your own risk:
 - [Ansible role using Supervisor](https://github.com/astorije/ansible-thelounge)
 - [ARMHF Docker images](https://hub.docker.com/r/lsioarmhf/thelounge/)
 - [YunoHost app](https://github.com/YunoHost-Apps/thelounge_ynh)
+- [Cloudron app](https://cloudron.io/store/io.github.thelounge.html) [Source](https://git.cloudron.io/cloudron/thelounge-app)
```
Comment (xPaw) [inline _docs/install-and-upgrade.md:205]: Can you remove Source? Looks odd in text.

File: _docs/install-and-upgrade.md [hunk 2]
```diff
@@ -211,7 +212,6 @@ configuration to get up and running:
 
 {:.app-stores}
 [![Bytesized Hosting logo](/img/logos/bytesized-hosting.svg)](https://bytesized-hosting.com/ "Bytesized Hosting")
-[![Cloudron logo](/img/logos/cloudron.svg)](https://cloudron.io/button/io.github.thelounge.html "Cloudron")
```
Comment (xPaw) [inline _docs/install-and-upgrade.md:214]: Can you delete the image file too please.

File: _docs/install-and-upgrade.md (hunk 1 above)
Comment (gramakri) [reply _docs/install-and-upgrade.md:205]: @xPaw will do

File: _docs/install-and-upgrade.md (hunk 2 above)
Comment (gramakri) [reply _docs/install-and-upgrade.md:214]: @xPaw done

**Pull request merged**